system/Database/Postgre/PreparedQuery.php

```diff
@@ -98,6 +98,14 @@ public function _getResult()
         return $this->result;
     }
 
+    /**
+     * Deallocate prepared statements
+     */
+    protected function _close(): bool
+    {
+        return (bool) pg_query($this->db->connID, 'DEALLOCATE "' . $this->db->escapeIdentifiers($this->name) . '"');
```

Comment:
I prefer to see this checked against its specific return type.
```suggestion
        return pg_query($this->db->connID, 'DEALLOCATE "' . $this->db->escapeIdentifiers($this->name) . '"') !== false;
```

Reply:
I've changed this
But other methods in Postgre connector are still using the cast to boolean (eg in Database\Postgre\Connection)

Reply:
You can send another PR to refactor the code, if you like.